packages/serverless/src/model/database/group.ts

```diff
@@ -35,6 +36,14 @@ export class ModelGroup extends ModelGeneral<OptionalId<TGroupRecord>> {
       await addTimestampMongoDB(collection, session);
     }
   }
+
+  /** Drop group and all relations of user in group. */
+  public async dropGroup(groupObjectId: ObjectId, session: ClientSession) {
+    const imUserGroup = new ModelUserGroup(this.databaseName);
+
+    await imUserGroup.deleteMany({ groupObjectId }, session);
+    await this.deleteOne({ _id: groupObjectId }, session);
```

Comment:
```suggestion
    await imUserGroup.deleteMany({ groupObjectId }, { session });
    await this.deleteOne({ _id: groupObjectId }, {session});
```
Or does it have an overloading version for session?

Reply:
fixed